Mathlib/MeasureTheory/Measure/Hausdorff-orthogonal-projection.lean

```diff
@@ -0,0 +1,98 @@
+/-
+Copyright (c) 2025 Francesco Nishanil Chotuck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Francesco Nishanil Chotuck
+-/
+
+import Mathlib.MeasureTheory.Measure.Haar.Disintegration
+import Mathlib.MeasureTheory.Measure.Hausdorff
+import Mathlib.Order.CompletePartialOrder
+
+/-!
+# Orthogonal Projections and Hausdorff Measure
+
+This file proves that Hausdorff measure is non-increasing under orthogonal projection
+onto a finite-dimensional Euclidean subspace.
+
+## Main results
+
+* `hausdorffMeasure_orthogonalProjection_le`: The `s`-dimensional Hausdorff
+  measure of the projection of a set is at most the measure of the original set.
+
+## References
+
+This formalisation is based on Lemma 3.5 from:
+
+- Julian Fox, *Besicovitch Sets, Kakeya Sets, and Their Properties*,
+  University of Chicago REU, 2021.
+  https://math.uchicago.edu/~may/REU2021/REUPapers/Fox.pdf
+-/
+
+open ENNReal MeasureTheory
+
+/--
+In a finite-dimensional Euclidean space, the norm of the orthogonal projection
+of a vector `v` onto a subspace `V` is less than or equal to the norm of `v`.
+-/
+lemma norm_orthogonalProjection_le (n : ℕ)
```

Comment:
I suggest putting this lemma just after [Submodule.orthogonalProjection_norm_le](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/InnerProductSpace/Projection.html#Submodule.orthogonalProjection_norm_le)